internal/extproc/server.go

```diff
@@ -63,39 +61,9 @@ func NewServer(logger *slog.Logger, tracing tracing.Tracing) (*Server, error) {
 
 // LoadConfig updates the configuration of the external processor.
 func (s *Server) LoadConfig(ctx context.Context, config *filterapi.Config) error {
-	backends := make(map[string]*processorConfigBackend, len(config.Backends))
```

Comment:
note: mainly this PR is moving this logic and the processorConfig* structs into filterapi/runtimefc package. Others are mechanical